libpysal/graph/base.py

```diff
@@ -2651,6 +2651,10 @@ def describe(
         if (y.index != self.unique_ids).all():
             raise ValueError("The values index is not aligned with the graph index.")
 
+        # reset numerical index to enable numba functionality
+        if isinstance(y.index.dtype, object):
+            y.index = np.arange(len(y.index))
```

Comment:
```suggestion
            y = y.reset_index(drop=True)
```
this may be easier as with the default RangeIndex you don't actually have to create the array.